spec/01-abstract.md

```diff
@@ -1,19 +1,39 @@
-## Problem Statement
+# Problem Statement
 
-Distributed tracing is a methodology implemented by tracing tools enabling to follow, analyze and debug a transaction across multiple software components. Typically, a transaction passes more than one component which requires a transaction to be uniquely identified across all software components. Passing along this unique identification is referred to as context propagation.
+Distributed tracing is a methodology implemented by tracing tools to follow, analyze
+and debug a transaction across multiple software components. Typically, a trace
+traverses more than one component which requires it to be uniquely identifiable
+across all participating systems. Passing along this unique identification is
+referred to as *context propagation*.

```

Comment:
it's a _job_ of context propagation, it's not _the_ context propagation - the latter can exist and be useful without unique identifier.

Reply:
Fixed with #244